### PR TITLE
TagBox: remove tag button click should not remove tags in other TagBoxes with fieldTemplate (T1137828)

### PR DESCRIPTION
**Pull request by Zedwag**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js [hunk 1]
```diff
@@ -5068,6 +5068,33 @@ QUnit.module('the \'fieldTemplate\' option', moduleSetup, () => {
         assert.deepEqual($tagBox.dxTagBox('option', 'value'), [], 'value was cleared');
         assert.equal($field.text(), '', 'text was cleared after the deselect');
     });
+
+    QUnit.module('one tabBox with fieldTemplate and another tagBox', {
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js]:
```suggestion
    QUnit.module('one tgbBox with fieldTemplate and another tagBox', {
```

File: testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js [hunk 2]
```diff
@@ -5068,6 +5068,33 @@ QUnit.module('the \'fieldTemplate\' option', moduleSetup, () => {
...
+                value: [1],
+            });
+            this.tagBox = $tagBox.dxTagBox('instance');
+
+            this.tagBoxWithFieldTemplate = $('#tagBox').dxTagBox({
+                items: [1, 2, 3],
+                value: [1],
+                fieldTemplate: () => $('<div>').dxTextBox()
+            }).dxTagBox('instance');
+
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js]:
```suggestion
```

File: testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js [hunk 3]
```diff
@@ -5068,6 +5068,33 @@ QUnit.module('the \'fieldTemplate\' option', moduleSetup, () => {
...
+
+            $tagBox.find('.dx-tag-remove-button').trigger('dxclick');
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js]: let's use `TAGBOX_TAG_REMOVE_BUTTON_CLASS` constant

File: testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js [hunk 4]
```diff
@@ -5068,6 +5068,33 @@ QUnit.module('the \'fieldTemplate\' option', moduleSetup, () => {
...
+        }
+    }, () => {
+        QUnit.test('tag should be removed after click on remove button in tagBox without template (T1137828)', function(assert) {
+            assert.strictEqual(this.tagBox.option('value').length, 0);
+        });
+
+        QUnit.test('no tags should be removed in tagBox with template after removing tag in tagBox without template (T1137828)', function(assert) {
+            assert.strictEqual(this.tagBoxWithFieldTemplate.option('value').length, 1);
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js]: I don't see a reason to separate this tests. But see strange moments:
1. test = assert now
2. test does not include action. Click is only in beforeEach where we usually only prepare a data
3. module name is really strange :) I don't think new tests can be added here in the future

Let's just write the single usual test?

Comment (Zedwag) [reply testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js]: done

**Review by ksercs — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**